### PR TITLE
Update Kenya Nandi county crop type project to use segmentation

### PR DESCRIPTION
**Pull request by favyen2**
This depends on changes in https://github.com/allenai/rslearn/pull/223

Remaining things to do:
* Use the padding from the RasterMerger. The main benefit is that the predictions won't be offset (currently the prediction at each 4x4 input shows up in the output at the topleft pixel of that 4x4, but really it should be more centered), since anyway the underlying model outputs the same class at each pixel.

### CODE REVIEW

File: rslp/crop/kenya_nandi/train.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+        This only works when all of the pixels have the same segmentation target.
+        """
+        output_probs = super().forward(features, inputs)
+        # BC -> BCHW
+        h, w = inputs[0]["sentinel2_l2a"].shape[1:3]
+        return output_probs[:, :, None, None].repeat([1, 1, h, w])
+
+
+class MultiSegmentationPoolingDecoder(torch.nn.Module):
```
Comment (yawenzzzz) [inline rslp/crop/kenya_nandi/train.py:26]: What is this one used for?
